sims/verilator/Makefile

```diff
@@ -1,112 +1,175 @@
 # general path variables
-#########################################################################################
+#============================================================================
 base_dir=$(abspath ../..)
 sim_dir=$(abspath .)
 
-#########################################################################################
+#============================================================================
 # include shared variables
-#########################################################################################
+#============================================================================
 include $(base_dir)/variables.mk
 
-#########################################################################################
+#============================================================================
 # name of simulator (used to generate *.f arguments file)
-#########################################################################################
+#============================================================================
 sim_name = verilator
 
-#########################################################################################
-# vcs simulator types and rules
-#########################################################################################
+#============================================================================
+# verilator simulator types and rules
+#============================================================================
 sim_prefix = simulator
 sim = $(sim_dir)/$(sim_prefix)-$(MODEL_PACKAGE)-$(CONFIG)
 sim_debug = $(sim_dir)/$(sim_prefix)-$(MODEL_PACKAGE)-$(CONFIG)-debug
 
-PERMISSIVE_ON=
-PERMISSIVE_OFF=
-
-WAVEFORM_FLAG=-v$(sim_out_name).vcd
-
 .PHONY: default debug
 default: $(sim)
 debug: $(sim_debug)
 
-#########################################################################################
+#============================================================================
 # import other necessary rules and variables
-#########################################################################################
+#============================================================================
 include $(base_dir)/common.mk
 
-#########################################################################################
-# verilator binary and flags
-#########################################################################################
+#============================================================================
+# selectively enabling printfs
+#============================================================================
+ENABLE_PRINTF_PATTERN ?=
+sim_vsrcs_printf := $(sim_vsrcs:.v=.printf.v)
+
+#============================================================================
+# verilator/cxx binary and flags
+#============================================================================
+# can't use -Wall, firrtl spits out 1000's of warnings that flag as errors
+#VERILATOR := verilator --cc --exe -Wall
```

Comment:
Can you delete all commented out lines like this? Also `-Wall` is added later anyways... and I recently added `-Wno-fatal` which should make this comment incorrect. Right?

Reply:
where is -Wall added? i'm pretty sure it isn't.

the -Wno-fatal just makes verilator not abort when warning are emitted. my goal is to just not print all the warnings since firrtl produces too many. this is not ideal, but i can't change firrtl-emitted code.

Reply:
Ahh sorry, it is just added when building Ariane. Im fine not printing FIRRTL errors, but maybe someone else knows why it was added by default.

Reply:
i removed the unnecessary comments